packages/@sanity/block-tools/package.json

```diff
@@ -49,17 +49,17 @@
     "watch": "pkg-utils watch"
   },
   "dependencies": {
+    "@sanity/types": "3.52.2",
+    "@types/react": "^18.3.3",
```

Comment:
This package has a dependency on `react` typings:
```ts
import {type ComponentType} from 'react'
```

The pattern established by `@sanity/types` is to declare `@types/react` in `dependencies` instead of `devDependencies`